src/fides/api/models/privacy_request.py

```diff
@@ -1307,6 +1307,7 @@ class ProvidedIdentityType(EnumType):
     ga_client_id = "ga_client_id"
     ljt_readerID = "ljt_readerID"
     fides_user_device_id = "fides_user_device_id"
+    external_id = "external_id"
```

Comment:
I don't think this enum is used in the database but can you double check if you haven't?

Reply:
excellent